### PR TITLE
HAI-CA Examples

### PR DESCRIPTION
**Pull request by astevensgtri**

### CODE REVIEW

**Review by myungchoi — APPROVED** [review]
we would want to add the display for the codes. And, we would want to change from OID to HTTP notation for the system in codeableconcept.

File: fsh-generated/resources/Condition-HAICA-TC-Condition.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ]
+  },
+  "extension": [
+    {
+      "url": "http://cbsig.chai.gatech.edu/StructureDefinition/cbs-case-class-status",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "code": "410605003",
+            "system": "http://snomed.info/sct"
```
Comment (EllieShivers) [inline fsh-generated/resources/Condition-HAICA-TC-Condition.json:16]: Display needed

File: fsh-generated/resources/Encounter-HAICA-TC-Hospitalization.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "id": "HAICA-TC-Hospitalization",
+  "meta": {
+    "profile": [
+      "http://cbsig.chai.gatech.edu/StructureDefinition/cbs-hospitalization"
+    ]
+  },
+  "status": "unknown",
+  "class": {
+    "code": "IMP",
+    "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode"
```
Comment (EllieShivers) [inline fsh-generated/resources/Encounter-HAICA-TC-Hospitalization.json:12]: Display needed

Comment (EllieShivers) [inline fsh-generated/resources/Encounter-HAICA-TC-Hospitalization.json:12]: Display

**Review by EllieShivers — APPROVED** [review]

**Pull request merged**